config/vsphere-csi-driver/overwrites/values.yaml

```diff
@@ -54,6 +22,10 @@ storageClass:
     vcdStorageProfileName: "vSAN Default Storage Policy"
     fileSystem: "ext4"
 
+internalFeatureStates:
```

Comment:
I think it's better to have this configurable and under that key in case others arise.
Also we might have to reconsider this setting at some point (not sure if we need it or not).
[Erkan+Jiri thread here](https://gigantic.slack.com/archives/C01BYMF6RN0/p1678283296261439?thread_ts=1678278670.181549&cid=C01BYMF6RN0)